env.example

```diff
@@ -351,3 +354,40 @@ RESTART_POLICY=unless-stopped
 
 # Authenticate using external service or just focus external auth window if there is one already.
 # TOKEN_AUTH_URL=https://auth.meet.example.com/{room}
+
+# Jicofo config
+JICOFO_SHORT_ID=1
+
+# The bridge selection strategy. The built-in strategies are:
+# --> SingleBridgeSelectionStrategy: Use the least loaded bridge, do not split a conference between bridges (Octo).
+# --> SplitBridgeSelectionStrategy: Use a separate bridge for each participant (for testing).
+# --> RegionBasedBridgeSelectionStrategy: Attempt to put each participant in a bridge in their local region (i.e. use Octo for geo-location).
+# --> IntraRegionBridgeSelectionStrategy: Use additional bridges when a bridge becomes overloaded (i.e. use Octo for load balancing). 
+OCTO_BRIDGE_SELECTION_STRATEGY=SingleBridgeSelectionStrategy
+
+# Configure the codecs and RTP extensions to be used in the offer sent to clients.
+ENABLE_CODEC_VP8=true
+ENABLE_CODEC_VP9=false
```

Comment:
@bgrozev I have disabled VP9 for now. I'm seeing some issues when a meeting is started and one of the participant is using Mozilla firefox and other is using chrome. video started freezing on firefox and finall stopped. Chrome to Edge, Brave or vice-versa conferences are working fine as they all are built on top of chromium.

Reply:
**Mozilla VP9 issue**
https://bugzilla.mozilla.org/show_bug.cgi?id=1633876